Running "bowl performance" is not straightforward

See also #41.

From my reading of the README I should be able to say `bowl performance` from `/var/apps/pp-development` to bring up a full stack.

The readme says "`Install dependencies for each required app in /var/apps by following the instructions in their README files`" but it isn't clear which apps are "required", and the indirection makes it more work than necessary to get started.

I first encountered `12:22:54 stagecraft.1              | /var/apps/stagecraft/start-app.sh: line 16: /home/vagrant/.virtualenvs/stagecraft/bin/activate: No such file or directory` which @paulfurley got me past by suggesting that I deleted `.virtualenvs/stagecraft` but now I'm stuck at:

```
$ bowl performance
I, [2014-03-05T13:27:36.003748 #24980]  INFO -- : Starting screenshot_as_a_service, spotlight, data_out, backdrop_read, backdrop_write, backdrop_admin, backdrop, stagecraft, performance..
13:27:36 screenshot_as_a_service.1 | started with pid 24983
13:27:36 spotlight.1               | started with pid 24985
13:27:36 backdrop_read.1           | started with pid 24988
13:27:36 backdrop_write.1          | started with pid 24990
13:27:36 backdrop_admin.1          | started with pid 24994
13:27:36 stagecraft.1              | started with pid 24997
13:27:36 spotlight.1               | Installing Node.js and Ruby dependencies...
13:27:36 backdrop_read.1           | waiting for startup lock
13:27:36 backdrop_write.1          | waiting for startup lock
13:27:36 backdrop_admin.1          | waiting for startup lock
13:27:36 screenshot_as_a_service.1 | 
13:27:36 screenshot_as_a_service.1 | module.js:340
13:27:36 screenshot_as_a_service.1 |     throw err;
13:27:36 screenshot_as_a_service.1 |           ^
13:27:36 screenshot_as_a_service.1 | Error: Cannot find module 'config'
13:27:36 screenshot_as_a_service.1 |     at Function.Module._resolveFilename (module.js:338:15)
13:27:36 screenshot_as_a_service.1 |     at Function.Module._load (module.js:280:25)
13:27:36 screenshot_as_a_service.1 |     at Module.require (module.js:364:17)
13:27:36 screenshot_as_a_service.1 |     at require (module.js:380:17)
13:27:36 screenshot_as_a_service.1 |     at Object.<anonymous> (/var/apps/screenshot-as-a-service/app.js:4:14)
13:27:36 screenshot_as_a_service.1 |     at Module._compile (module.js:456:26)
13:27:36 screenshot_as_a_service.1 |     at Object.Module._extensions..js (module.js:474:10)
13:27:36 screenshot_as_a_service.1 |     at Module.load (module.js:356:32)
13:27:36 screenshot_as_a_service.1 |     at Function.Module._load (module.js:312:12)
13:27:36 screenshot_as_a_service.1 |     at Function.Module.runMain (module.js:497:10)
13:27:36 screenshot_as_a_service.1 | exited with code 8
13:27:36 system                    | sending SIGTERM to all processes
13:27:36 spotlight.1               | terminated by SIGTERM
13:27:36 backdrop_read.1           | terminated by SIGTERM
13:27:36 backdrop_write.1          | terminated by SIGTERM
13:27:36 backdrop_admin.1          | terminated by SIGTERM
13:27:36 stagecraft.1              | terminated by SIGTERM
```

--- a/Procfile
+++ b/Procfile
@@ -1,4 +1,4 @@
-screenshot_as_a_service: ./run_in.sh ../screenshot-as-a-service node app.js
+screenshot_as_a_service: ./run_in.sh ../screenshot-as-a-service ./start-app.sh
 spotlight: ./run_in.sh ../spotlight ./start-app.sh
 backdrop_read: ./run_in.sh ../backdrop ./start-app.sh read 3038
 backdrop_write: ./run_in.sh ../backdrop ./start-app.sh write 3039

